core/vm/runtime/impl/runtime_impl.cpp

```diff
@@ -214,6 +216,31 @@ namespace fc::vm::runtime {
     return proofs::Proofs::verifyWindowPoSt(preprocess_info);
   }
 
+  outcome::result<std::map<Address, std::vector<bool>>>
+  RuntimeImpl::verifyBatchSeals(
+      const adt::Map<adt::Array<SealVerifyInfo>, adt::AddressKeyer> &seals) {
+    std::map<Address, std::vector<bool>> res;
+    OUTCOME_TRY(seals.hamt.visit(
```

Comment:
Why not use typed `seals.visit`?

Reply:
ok

Reply:
Fixed as adt::Map::visit is constant now.